projects/angular/src/button/_properties.buttons.scss

```diff
@@ -137,28 +137,28 @@
 
       // Warning button colors
       --clr-btn-warning-color: var(--clr-color-neutral-0);
```

Comment:
```suggestion
      --clr-btn-warning-color: var(--clr-btn-warning-color);
```

Reply:
@dtsanevmw I don't think `--clr-btn-warning-color` referencing itself would help. 
Also all of the button colors are using `--clr-color-neutral-0` for that variable.
What is your logic behind the suggestion?

Reply:
@valentin-mladenov Sorry I reviewed in the demo app and there it was overridden by the shim.cds-core. However all the button seems to use neutral-0 except the warning button, that's what I see in core. I suspect it's because of the contrast ratio for a11y. The right one seems to be that `--clr-btn-warning-color: var(--cds-global-color-construction-900);` which is equivalent to `var(--clr-color-neutral-900)`. 
![Screenshot 2023-05-29 at 10 46 39](https://github.com/vmware-clarity/ng-clarity/assets/127101685/0ae1312b-fff0-4992-999c-0c1524ca89ab)

Reply:
Changed to use neutral 900 in commit e1c6bd5

Reply:
After some more investigation the color `--cds-global-color-construction-900` used for warning in core, doesn't look correct. When switching to high contrast theme the text blends with the background of the button.
https://storybook.core.clarity.design/?path=/docs/components-button--page&globals=theme:high-contrast

@colinreedmiller which color shall we use `--clr-color-neutral-0` or `--clr-color-neutral-900`?